frontend/packages/dev-console/src/components/import/builder/SelectorCard.tsx

```diff
@@ -4,22 +4,22 @@ import { Card, CardHeader, CardBody } from '@patternfly/react-core';
 import { StarIcon } from '@patternfly/react-icons';
 import { BuilderImage } from '../../../utils/imagestream-utils';
 
-import './BuilderImageCard.scss';
+import './SelectorCard.scss';
 
-export interface BuilderImageCardProps {
+export interface SelectorCardProps {
   image: BuilderImage;
   selected: boolean;
-  recommended: boolean;
+  recommended?: boolean;
   onChange: (name: string) => void;
 }
 
-const BuilderImageCard: React.FC<BuilderImageCardProps> = ({
+const SelectorCard: React.FC<SelectorCardProps> = ({
```

Comment:
Would be useful as a reusable component along with the parent container. Maybe even a formik field.

Reply:
yeah, parent has lot of code which are coupled as it's used in git import and catalog

Reply:
Yeah, I like the idea of having a `ImageSelectorField` that wraps all the logic and gives a clean API to use. It was not a shared component as there was no other use case for it till now. I had a discussion with Jai around this today and we came to a solution that works for both use cases.

Reply:
yeah , have updated the code PTAL :)